pkg/cmd/clustertriggerbinding/list.go

```diff
@@ -98,7 +103,7 @@ or
 	}
 
 	f.AddFlags(c)
-
+	c.Flags().BoolVarP(&opts.NoHeaders, "no-headers", "", opts.NoHeaders, "do not print column headers with output (default print column headers with output)")
```

Comment:
You should be able to just use `BoolVar` since there is no shorthand:

```suggestion
	c.Flags().BoolVar(&opts.NoHeaders, "no-headers", opts.NoHeaders, "do not print column headers with output (default print column headers with output)")
```

Reply:
Okay.